recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,2 @@
 channel_targets:
-  - conda-forge cdat_dev
-channel_sources:
-  - conda-forge/label/cdat_dev,conda-forge,defaults
+  - conda-forge cdms2_dev
```

Comment:
I'm changing the dev label for convenience -- I don't want to bring in all packages from the `cdat_dev` label, just this one.